voter-dapp/src/ActiveRequests.js

```diff
@@ -117,6 +122,23 @@ function ActiveRequests() {
   }, [subsequentFetchComplete, voteStatusesStringified, decryptionKeys, account]);
   const decryptionComplete = decryptedCommits && voteStatuses && decryptedCommits.length === voteStatuses.length;
 
+  const { send: batchRevealFunction, status } = useCacheSend("Voting", "batchReveal");
+  const onClickHandler = () => {
+    const reveals = [];
+    for (const index of Object.keys(checkboxesChecked)) {
```

Comment:
nit: isn't this the same as `for (const index in checkboxesChecked)`?

Reply:
Done.